src/main/scala/nl.knaw.dans.easy.deposit/StateManager.scala

```diff
@@ -29,33 +29,41 @@ import scala.util.{ Failure, Success, Try }
 case class StateManager(depositDir: File, submitBase: File) {
   private val stateDescriptionKey = "state.description"
   private val stateLabelKey = "state.label"
+
+  /** created/destroyed when the state changes, needed to peek into SUBMITTED/UUID/deposit.properties */
   private val bagIdKey = "bag-store.bag-id"
+
+  /** Available as side effect for properties not related to fetching or updating the state of the deposit. */
   val draftProps = new PropertiesConfiguration(
     (depositDir / "deposit.properties").toJava
   )
   private lazy val submittedProps = new PropertiesConfiguration(
     (submitBase / getProp(bagIdKey, draftProps) / "deposit.properties").toJava
   )
 
+  /** @return the state-label/description from drafts/USER/UUID/deposit.properties
+   *          unless more recent values might be available in SUBMITTED/UUID/deposit.properties
+   */
   def getStateInfo: Try[StateInfo] = Try {
-    val draftState = getStateLabel(draftProps)
-    draftState match {
+    getStateLabel(draftProps) match {
       case State.submitted | State.inProgress =>
         val newState: StateInfo = getProp(stateLabelKey, submittedProps) match {
           case "SUBMITTED" => StateInfo(State.submitted, getStateDescription(draftProps))
           case "REJECTED" => StateInfo(State.rejected, getStateDescription(submittedProps))
-          case "FEDORA_ARCHIVED" => StateInfo(State.archived, "The dataset is published")
-          case "IN_REVIEW" => StateInfo(State.inProgress, "The dataset is visible for you under your datasets in https://easy.dans.knaw.nl/ui")
-          case "FAILED" => StateInfo(State.inProgress, "The dataset is in progress")
+          case "FEDORA_ARCHIVED" => StateInfo(State.archived, s"The dataset is published at $landingPage")
```

Comment:
if we can return the URL as a separate JSON field in these cases that currently have the `$landingPage`, we can render that in an `onClick` on that row. In that case we don't require this `$landingPage` here...

Reply:
That would mean extending the models for StatusInfo and DepositInfo, perhaps another pull request? Merging will need synchronisation with the UI.
https://github.com/DANS-KNAW/easy-deposit-api/blob/5ebe7da4d3c451099c73b2cff1b6e2b13ac97f30/docs/api/api.yml#L723-L755